packages/tables/backend/sirius-components-tables/src/main/java/org/eclipse/sirius/components/tables/CheckboxCell.java

```diff
@@ -123,13 +132,19 @@ public Builder value(boolean value) {
             return this;
         }
 
+        public Builder descriptionId(String descriptionId) {
+            this.descriptionId = descriptionId;
+            return this;
+        }
```

Comment:
Do not tolerate null here, the same comment can be applied to half a dozens other locations.